app/services/host_inventory_api.rb

```diff
@@ -50,6 +50,12 @@ def system_profile(ids)
     end
   end
 
+  def hosts
+    JSON.parse(Platform.connection.get(
+      @url, {}, X_RH_IDENTITY: @b64_identity
```

Comment:
Nit: Can this use named attributes, at it is not immediately obvious on what is `{}` for?